exercises/grade-school/src/test/java/SchoolTest.java

```diff
@@ -29,7 +28,7 @@ public void addingAStudentAddsThemToTheSortedRoster() {
     school = new School();
     school.add("Aimee", 2);
     List<String> expected = Arrays.asList("Aimee");
-    assertThat(school.roster(), is(expected));
+    assertEquals(expected,school.roster());
```

Comment:
There should be a space here between `expected` and `school.roster()` 🙂

Reply:
all done